src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -369,7 +369,7 @@
   - name: --network-plugin
     type: string
     short-summary: The Kubernetes network plugin to use.
-    long-summary: Specify "azure" for advanced networking configurations. Defaults to "kubenet".
+    long-summary: Specify "azure" for routable pod IPs from VNET, "kubenet" for non-routable pod IPs with an overlay network, or "none" for no networking configured. Defaults to "kubenet".
```

Comment:
Missing help message update several lines above. Opened another PR [#23344](https://github.com/Azure/azure-cli/pull/23344) to catch the July release.